Ago-Dic-2020/salazar-coronado-juan-daniel/practica2/practica2.py

```diff
@@ -0,0 +1,71 @@
+class Estudiante():
+    def __init__(self, nombre, edad, semestre, carrera):
+        self.nombre = nombre
+        self.edad = edad
+        self.semestre = semestre
+        self.carrera = carrera
+
+    def presentacion(self):
+        return "Soy " + self.nombre + ", tengo " + self.edad + " años y estoy en " + self.semestre + " semestre."
+
+    def getCarrera(self):
+        carreras = {
+        'ISC': "Ing. En sistemas computacionales",
+        'IIS': "Ing. Industrial y de sistemas",
+        'IEC': "Ing. En electronica y comunicaciones",
+        'ITC': "Ingeniero en tecnología y comunicaciones",
+        'LSA': "Lic. En sistemas computacionales y administrativos",
+        'IA': "Ingeniero automotriz"
+        }
+        return "Estudio la carrera de " + carreras.get(self.carrera)
+    
+    def MotivacionPromedio(self, promedio):
+        if promedio >= 70 and promedio <= 79:
+            return "necesito esforzarme más"
+        elif promedio > 79 and promedio <= 89:
+            return "lo estoy haciendo bien, pero puedo hacerlo mejor"
+        elif promedio >= 90 and promedio <= 100:
+            return "voy muy bien, debo seguir así"
+        else:
+            return "-> promedio fuera de los parametros normales <-"
+
+class ServicioSocialYPracticas(Estudiante):
+    def __init__(self, nombre, edad, semestre, carrera, servicio, practicas):
+        Estudiante.__init__(self, nombre, edad, semestre, carrera)
+        self.servicio = servicio
+        self.practicas = practicas
+        
+    def Estado(self):
+        if self.servicio == True and self.practicas == True:
+            return "Ya realice el servicio social y las practicas profesionales"
+        elif self.servicio == False and self.practicas == False:
```

Comment:
Mismo comentario sobre utilizar el valor de los booleanos como default para las evaluaciones lógicas, solamente que en este caso tendríamos que utilizar la negación, por estar evaluando un valor `False`, es decir `if self.servicio == False and self.practicas == False:` es igual a `if not self.servicio and not self.practicas:`